client/src/Components/Header.jsx

```diff
@@ -53,40 +39,37 @@ const Header = () => {
 				}}
 				px={{ xs: 2, md: 5 }}
 			>
-				<Tabs
-					value={value}
-					onChange={handleChange}
-					sx={{
-						display: "flex",
-						justifyContent: "space-evenly",
-						borderColor: "divider",
-						flexGrow: 1,
-					}}
-					TabIndicatorProps={{ style: { backgroundColor: "#fff" } }}
-					selectionFollowsFocus
-				>
-					<Tab
-						label="BOOKME"
-						sx={{
-							fontWeight: "bolder",
-							fontSize: 25,
-							color: "#000",
-						}}
-					/>
-					<Tab label="Find" sx={{ color: "#000" }} />
-					<Tab label="Book" sx={{ color: "#000" }} />
+				<Box sx={{ display: "flex", justifyContent: "space-evenly" }}>
+					<MenuItem component={Link} to="/">
+						<Typography
+							variant="h5"
+							sx={{
+								fontWeight: "bolder",
+								cursor: "pointer",
+							}}
+						>
+							BOOKME
+						</Typography>
+					</MenuItem>
+					<MenuItem component={Link} to="/find" fontWeight="bold">
+						<Typography fontWeight="bolder">Find</Typography>
+					</MenuItem>
+					<MenuItem component={Link} to="/find" fontWeight="bold">
+						<Typography fontWeight="bolder">Book</Typography>
+					</MenuItem>
```

Comment:
Find and Book go to the same url `/find`. Is this correct?

Reply:
yes, because at the moment we didn't have any page for Book so I thought it is better that empty.